tests/Backend/Workspaces/Backend/WorkspaceBackendFactory.php

```diff
@@ -30,8 +30,6 @@ public static function createWorkspaceBackend(array $workspace, bool $useDBAL =
                 return new SynapseWorkspaceBackend($workspace);
             case StorageApiTestCase::BACKEND_EXASOL:
                 return new ExasolWorkspaceBackend($workspace);
```

Comment:
A tohle ne?

A to pod tím ano?

Reply:
imho je to tak spravne. Ta strategie byla "teradata se nikdy nevrati (to fakt vime, neslo to, nefungovalo to, takze je to fakt mrtvej kod). Exasol se muze klidne vratit, protoze to technicky fungovalo, jen o to nebyl businessovy zajem

Reply:
ok